.gitignore

```diff
@@ -54,3 +54,10 @@ config/*.env
 .eclipse/
 .elixir_ls/
 /.project
+assets/package-lock.json
```

Comment:
Could you revert these changes? I don't think they're necessary.

Reply:
Done. Learning as I go :)